chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,869 @@
+    const text1 = (text || '').trim();
+    const text1lc = text1.toLowerCase();
+    const href1 = (href || '').trim();
+    const href1lc = href1.toLowerCase();
+    const comment1 = (comment || '').trim();
+
+    const retn = '';
+
+    if (text1.length < 1 && href1.length < 1) {
+      // Intetionally blank
+    } else if (text1lc === href1lc) {
+      retn = ` <${href1}> `; // This renders correctly in Trello as a sole URL
```

Biome (1.9.4)

[error] Can't assign retn because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)